submissions/oluwabusolamiEluyela/html-forms/contact.html

```diff
@@ -5,14 +5,19 @@
     <title>Contact Us!</title>
 </head>
 <body>
-<nav>
+<header>
     <h1>JUSTIFY</h1>
+    <nav>
     <a href="../html/index.html">Product Page</a>
     <a href="../html-forms/Contact.html">Contact Us</a>
```

Comment:
I can see Contact.html in your link, but the file name is contact.html. It won't navigate correctly. Kindly correct this. 

Also, you did not add navigation to your Product Page, kindly do that also.

Reply:
Done